### PR TITLE
[pt-BR] feature: started to translate learn/server-side/django/django_assessment_blog/index.html

### PR DESCRIPTION
**Pull request by igorkf**
This is a WIP PR.   
I started to translate [this](https://developer.mozilla.org/en-US/docs/Learn/Server-side/Django/django_assessment_blog) article from Django.      
Please, give any feedback if needed.

### CODE REVIEW

File: files/pt-br/learn/server-side/django/django_assessment_blog/index.html [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+   <td>Uma página inicial descrevendo o site.</td>
+  </tr>
+  <tr>
+   <td>Lista de todos os artigos do blog</td>
+   <td><code>/blog/blogs/</code></td>
+   <td>
+    <p>Lista de todos os artigos do blog:</p>
+
+    <ul>
+     <li>Accesível para todos os usuários através de um link na barra lateral.</li>
```
Comment (josielrocha) [inline files/pt-br/learn/server-side/django/django_assessment_blog/index.html:58]: Olá @igorkf, muito obrigado pela contribuição.
Acredito que aqui passou uma palavrinha despercebida.. `Accesivel` => `Acessível`.

Comment (igorkf) [reply files/pt-br/learn/server-side/django/django_assessment_blog/index.html:58]: Ops, obrigado pelo toque! Vou arrumar.

**Comment by SphinxKnight** [comment]
@igorkf / @mdn/yari-content-pt-br this PR is more than a year old and the content is now on Markdown? Should it be closed or be left as is?

**Comment by josielrocha** [comment]
Thank you for your help @igorkf!

This content has already been migrated to Mardown.

🚀

**Pull request closed without merging**